lara-typescript/src/interactive-api-parent/iframe-saver.ts

```diff
@@ -263,6 +266,10 @@ export class IFrameSaver {
       return this.getFirebaseJwt(request);
     });
 
+    this.addListener("getInteractiveList", (request: LaraInteractiveApi.IGetInteractiveListRequest) => {
```

Comment:
I think this iframe-saver is only used at runtime. The getInteractiveList api will be needed during authoring.